application/controllers/bestitamazoncron.php

```diff
@@ -220,6 +220,28 @@ protected function _updateRefundDetails()
         }
     }
 
+    /**
+     * Update suspended orders
+     * @throws oxSystemComponentException
+     * @throws oxConnectionException
+     */
+    protected function _closeOrders()
+    {
+        $aProcessed = $this->_processOrderStates(
+            "SELECT OXID, OXORDERNR FROM oxorder
+            WHERE BESTITAMAZONORDERREFERENCEID != ''
+              AND BESTITAMAZONAUTHORIZATIONID != ''
+              AND OXTRANSSTATUS = 'AMZ-Capture-Completed'",
+            'closeOrderReference'
+        );
+
+        foreach ($aProcessed as $sOrderNumber => $oData) {
+            if (isset($oData->CloseOrderReferenceResult) && isset($oData->ResponseMetadata->RequestId)) {
```

Comment:
You can use multiple parameters for isset:
e.g. in this case: 
` if (isset($oData->CloseOrderReferenceResult, $oData->ResponseMetadata->RequestId)) {`

Reply:
Nice!